src/Components/TrainingPage.js

```diff
@@ -1,21 +1,43 @@
-import React from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import React, { Component } from 'react';
+import { StyleSheet, ScrollView, Text } from 'react-native';
 import { connect } from 'react-redux';
+import DrillList from './DrillList';
 import theme from '../styles/theme.style';
 
+const getGoals = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  return Math.max(...nbPlayersList);
+};
+
+const getEquipment = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  return Math.max(...nbPlayersList);
+};
+
 const mapStateToProps = state => {
   return {
     drills: state.drills,
   };
 };
 
-export const TrainingPage = props => {
-  return (
-    <View style={styles.trainingPage}>
-      <Text>This will be a training page</Text>
-    </View>
-  );
-};
+class TrainingPage extends Component {
+  render() {
+    const { navigation } = this.props;
+    const training = this.props.route.params.training;
+    const drills = this.props.drills.filter(drill => training.drills.includes(drill.id));
+
+    return (
+      <ScrollView style={styles.trainingPage}>
+        <Text style={styles.descriptionText}>{training.description}</Text>
```

Comment:
Maybe add a training title?

Reply:
For now the training title is displayed in the header bar, as for drills, so I haven't put it again on the page. I think I will make another PR to improve the page design which is ugly for now.. I would be very happy to have some suggestions

Reply:
Yes that's right
I guess we'll have better ideas when the page is in the app for some time and we navigate it from time to time